app/assets/stylesheets/foundation-portfolio-theme.scss

```diff
@@ -1,7 +1,20 @@
-@import url('https://fonts.googleapis.com/css?family=Raleway:600,400,200');
+@import url('https://fonts.googleapis.com/css?family=Raleway:700,600,400,200');
 @import url('https://fonts.googleapis.com/css?family=Shadows+Into+Light');
 
 
+#david, #ker {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line